frontend/src/components/VGlobalAudioSection/VGlobalAudioSection.vue

```diff
@@ -147,3 +147,8 @@ export default defineComponent({
   },
 })
 </script>
+<style>
```

Comment:
I don't know if this is the style guide or a personal preference but there should be a blank line between `</script>` and `<style>`.